internal/controllers/nft_controller.go

```diff
@@ -837,3 +840,200 @@ func (nc *NFTController) GetVinCredential(c *fiber.Ctx) error {
 	c.Set(fiber.HeaderContentType, fiber.MIMEApplicationJSON)
 	return c.Send(nft.R.Claim.Credential.JSON)
 }
+
+// GetBurnDevice godoc
+// @Description Returns the data the user must sign in order to burn the device.
+// @Tags        user-devices
+// @Param       tokenID path int true "token id"
+// @Success     200          {object} apitypes.TypedData
+// @Security    BearerAuth
+// @Router      /vehicle/:tokenID/commands/burn [get]
+func (udc *UserDevicesController) GetBurnDevice(c *fiber.Ctx) error {
+	tis := c.Params("tokenID")
+	ti, ok := new(big.Int).SetString(tis, 10)
+	if !ok {
+		return fiber.NewError(fiber.StatusBadRequest, fmt.Sprintf("Couldn't parse token id %q.", tis))
+	}
+	tid := types.NewNullDecimal(new(decimal.Big).SetBigMantScale(ti, 0))
+
+	client := registry.Client{
+		Producer:     udc.producer,
+		RequestTopic: "topic.transaction.request.send",
```

Comment:
Should `topic.transaction.request.send` be a setting similar to some of our other topics.

Reply:
hesitant to do that bc that topic is used across rewards api, meta tx processor, etc... but it probably should be coming from settings at the very least rather than hardcoded !

Reply:
follow up q: if we made this change for this endpoint, we'd probably want to carry the change through to where this topic is published to elsewhere in the repo... should that be a separate PR making that change everywhere at once so that we don't clutter up this pr?